Makefile

```diff
@@ -1,2 +1,2 @@
-Constitution.pdf: Constitution.md Declaration.md Roles.md
+Constitution.pdf: Constitution.md appendices/A-declaration.md Roles.md appendices/*.md
```

Comment:
You sure this isn'tincluding A-declaration twice?

Reply:
it is, but `$^` only puts it in the constructed command line once (the first time it appears): https://www.gnu.org/software/make/manual/html_node/Automatic-Variables.html

> A target has only one prerequisite on each other file it depends on, no matter how many times each file is listed as a prerequisite.